### ISSUE TITLE
Reproducing full model's weights

### ISSUE THREAD

**Issue by guhur** [arjunmajum/vln-bert#10]
Hi Arjun,

I have some issues with reproducing your weights. I run the exact same commands as you wrote in your README.md. However, the performances are less good than yours. 

Here's the tensorboard of my success rate on val seen and val unseen when finetuning with ranking loss:

![SR on val seen and val unseen](https://user-images.githubusercontent.com/12297742/100545596-de20be80-325c-11eb-9c45-6270b09f5735.png)

One explanation could that the pre-training phase was not trained for enough time. On the following plots, it sounds as is 50 epochs is not enough.

![word masking loss](https://user-images.githubusercontent.com/12297742/100545646-1c1de280-325d-11eb-8228-2ff344b66602.png)
![image masking loss](https://user-images.githubusercontent.com/12297742/100545658-2cce5880-325d-11eb-9ed5-08b3e31b48c6.png)

Also, I wasn't testing with the exact same dependencies version, which might explain the differences. I am testing again with it.

Thanks for your guidance,
Pierre

**Comment by guhur** [comment]
Training with the same dependencies allowed me to reach almost the same accuracy (-0.2%)

**Comment by arjunmajum** [comment]
Awesome!

Would you mind posting you tensorboard curves so that people have your reference point?

**Comment by guhur** [comment]
Here are the TB curves for fine-tuning:

![image](https://user-images.githubusercontent.com/12297742/101325514-2bf48280-386c-11eb-9b32-55f2939e3d20.png)
